R/calc_match_addresses.R

```diff
@@ -44,7 +46,7 @@ calc_match_addresses <- function(
       suffix = c("", "_LOOKUP"),
       copy = TRUE
     ) %>%
-    dplyr::select(-JOIN_ADDRESS)
+    dplyr::select(-.data$JOIN_ADDRESS)
```

Comment:
This and all other changes in this file are just adding the `.data` pronoun to handle R CMD CHECK warnings

Reply:
...plus a missed namespacing of `across`